tools/autograd/derivatives.yaml

```
@@ -826,6 +826,17 @@
   self: at::_unsafe_index_put(grad.new_zeros_symint(self.sym_sizes(), self.options()), indices, grad, true)
   result: auto_linear
 
+- name: _unsafe_masked_index(Tensor self, Tensor mask, Tensor?[] indices, Scalar fill) -> Tensor
+  self: at::_unsafe_masked_index_put(grad.new_zeros_symint(self.sym_sizes(), self.options()), mask, indices, grad, true)
+  mask: non_differentiable
+  result: auto_linear
```

Almost! The correct line is:
```suggestion
  result: _unsafe_masked_index(self_t, mask, indices, 0)
```
since the derivative of the indices that we are filling in is 0 as they are constant.